lib/tasks/argo.rake

```diff
@@ -18,6 +18,7 @@ end
 task :default => [:ci]
 
 task :ci do
+  raise 'NEVER RUN CI ON PROD' if ENV['RAILS_ENV'] =~ /production/i
   ENV['RAILS_ENV'] = 'test'
```

Comment:
i'd need to double-check (and maybe do some testing) to be sure, but...

i think that by the time RAILS_ENV gets reset here, the environment specific configs would have already been loaded as part of Rails loading (e.g. production.rb, dor_production.rb, settings/production.yml).